draft-02.xml

```diff
@@ -969,10 +969,25 @@
             'safe', 'unsafe', or 'idempotent.'
           </t>
           <t>
-            The 'rt' attribute is OPTIONAL and is an opaque string and MAY match the
-            <xref target="prop-id" format="title"/>
-            of an
-            existing<xref target="prop-descriptor" format="title"/>.
+            The 'rt' attribute is OPTIONAL and SHOULD be one of the following:
+            <list style="numbers">
+              <t>
+                A fragment identifier (e.g. rt="#user") which points
+                to the <xref target="prop-id" format="title"/>
+                of an existing <xref target="prop-descriptor" format="title"/>
+                in the current ALPS document.
+              </t>
+              <t>
+                A resolvable URL (e.g. rt="http://example.org/profiles/user")
```

Comment:
I am not sure this one makes sense. I think this would have to point to a particular descriptor and thus this needs to be a local or absolute fragment. If you want to use this, I think you would need to but some extra verbage that says that the `rt` matches the same semantic element containing the `rt` declaration. E.g

```xml
<descriptor id="thing" rt="http://example.org/profiles/things"/>
```

Which would imply a descriptor 'thing' in the remote descriptor. I don't think that makes any sense and is not worth it. I think this just has to be a fragment pointing to an explicit descriptor. Otherwise it is meaningless. Make sense?